widget/embedded/src/components/Quote/Quote.styles.ts

```diff
@@ -112,6 +112,13 @@ export const SummaryContainer = styled('div', {
     paddingTop: '$10',
     display: 'flex',
     alignItems: 'center',
+    '.usd-value': {
+      $$color: '$colors$neutral600',
+      [`.${darkTheme} &`]: {
+        $$color: '$colors$neutral700',
+      },
+      color: '$$color',
+    },
   },
```

Comment:
Why usd value in input has a different color with output?


![Screenshot from 2023-11-19 08-43-18](https://github.com/rango-exchange/rango-client/assets/99180934/c0905df7-6c28-4d8d-a7f7-ce1995d8d844)

Reply:
fixed